static/js/analytics.js

```diff
@@ -140,6 +140,15 @@ const composeStatisticsFromSingleUserJson = (json) => {
   return pageHtml;
 };
 
+const composeGalleryLink = (userId) => {
+  const galleryLink = document.getElementById("gallery-analytics-link");
+  galleryLink.innerHTML = `
+  <a href="${baseUrl}/gallery/${userId}">
+      <button class="button header-button">Gallery</button>
```

Comment:
nit:  "Medias" instead of "Gallery"? 

Gallery is the page with all the users media, but from here we don't go the directory but to a specific user media

Reply:
Done